examples/triply/typedefs.h

```diff
@@ -40,14 +40,12 @@
 
 #ifdef EQUALIZER
 #  include <eq/eq.h>
-#  include <vmmlib/vmmlib.hpp>
 #  define PLYLIBASSERT  LBASSERT
 #  define PLYLIBERROR   LBERROR
 #  define PLYLIBWARN    LBWARN
 #  define PLYLIBINFO    LBINFO
 #else
 #  include <GL/glew.h>
-#  include <vmmlib/vmmlib.hpp>
 #  ifdef _WIN32
 #    include <Winsock2.h>
 #    include <Windows.h>
```

Comment:
Can you (re)move them in another commit, verifying the Windows build still works. They've been here for quite a bit.